code/index.html

```diff
@@ -3,28 +3,87 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=Karla:wght@300;400&display=swap" rel="stylesheet">
+    <title>Project 3 - Business Site</title>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+
+    <!-- Video hero header -->
+    <header>
+      <video playsinline muted loop autoplay>
+      <source src="videos/spa.mp4" type="video/mp4">
+      </video>
+
+      <section class="hero">
+        <h1>West coast night spa</h1>
+        <p>Book your experience now</p>
+      </section>
+    </header>
 
     <!-- Signup form -->
+    <section class="content">
+      <section class="form">
+      <h2>Request your booking here</h2>
+        <form action="http://httpbin.org/anything" method="post">
+          <label for="name">Name
+            <input class="input-box" name="name" id="name" type="text" placeholder="Enter your name" required>
+          </label>
+
+          <label for="email">E-mail
+            <input class="input-box" name="email" id="email" type="email" placeholder="Enter your e-mail" required>
+          </label>
+
+          <label for="date">Select date
+            <input class="input-box" type="date" name="bday" required pattern="\d{4}-\d{2}-\d{2}" required>
+            <span class="validity"></span>
+          </label>
+          
+          <div class="checkbox">
+            <input type="checkbox" id="check" name="check" value="check" required>
+            <label class="policy" for=check> I accept the <a href="#">terms</a> & <a href="#">conditions</a>
+            </label>
+          </div>
+          
+          <button class="submit-btn" type="submit">Request booking</button>
+        </form>
+      </section>
 
-    <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="accordion">
```

Comment:
Super nice accordion set up! I noticed you also used the small arrows in the questions boxes. Good job!